ceph/rados/rados_scrub.py

```diff
@@ -158,3 +158,21 @@ def add_begin_end_hours(self, begin_hours=0, end_hours=0):
             cluster_end_hour,
             cluster_end_weekday,
         )
+
+    def set_osd_flags(self, flag, value):
+        """
+        Used to set/unset the osd flags
+
+        Args:
+            1.flag: set or unset the flags
```

Comment:
Can we add examples of allowed values?

Reply:
I will add the example in the next PR which I need to raise after this merge.